config/ergw-gtp-c-node.config

```diff
@@ -42,7 +42,7 @@
        {[<<"apn02">>,<<"example">>,<<"net">>], [{vrf, upstream}]}
       ]},
 
-     {http_api, [{port, 8000}]}
+     {http_api, [{port, 8000}, {ip, {127, 0, 0, 1}}]}
```

Comment:
Yes. But @tlnd asked me to specify this as description that we can pass an `ip` here. Actually it should be documented in erGW in a better way though